### PR TITLE
feat: Add canonical links

### LINKED ISSUE #1545: Add canonical URLs to governance pages
```
<link rel="canonical" href="..." />
```

### PR DESCRIPTION
**Pull request by ncomerci**
**Screenshots**
image

**Related Issue**
This closes #1545 

- [ ]  **Functionality validation**
    - [ ]  Feature working as described in the related issue
    - [ ]  New environment variables have been set
    - [ ]  New API services methods have tests
    - [ ]  Code errors are being logged in the code health service
- [ ]  **Product validation**
    - [ ]  UI changes look the same as in Figma
    - [ ]  Usage events are being sent to the product analytics service
    - [ ]  New common components have been documented in Storybook
    - [ ]  Preview link if it’s a customer facing issue
- [ ] **Documentation**
    - [ ] README updated
    - [ ] New API changes are documented
- [ ] **Accessibility**
    - [ ] UI changes are accessible

### CODE REVIEW

File: src/components/Common/Head.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import Helmet, { HelmetProps } from 'react-helmet'
+
+import DclHead, { HeadProps } from 'decentraland-gatsby/dist/components/Head/Head'
+
+import { isLocalLink } from '../../clients/utils'
+import { GOVERNANCE_URL } from '../../constants'
+
+type Props = Partial<HeadProps> & HelmetProps
+
+function setAbsoluteUrl(links?: HelmetProps['link']) {
```
Comment (andyesp) [inline src/components/Common/Head.tsx:10]: `getAbsoluteUrl`

File: src/pages/transparency.tsx [hunk 2]
```diff
@@ -48,9 +48,10 @@ export default function TransparencyPage() {
     <>
       <Navigation activeTab={NavigationTab.Transparency} />
       <Head
-        title={t('page.transparency.title') || ''}
-        description={t('page.transparency.mission.description') || ''}
+        title={t('page.transparency.title')}
+        description={t('page.transparency.mission.description')}
         image="https://decentraland.org/images/decentraland.png"
+        link={[{ rel: 'canonical', href: locations.transparency() }]}
```
Comment (andyesp) [inline src/pages/transparency.tsx:54]: Shouldn't this prop be `links` as this can accept multiple ones?

Comment (ncomerci) [reply src/pages/transparency.tsx:54]: It is thus defined in Helmet's props, looking at the other props I think it refers to the name of the tag

image

File: src/clients/utils.ts [hunk 3]
```diff
@@ -33,3 +33,9 @@ export async function inBatches<T, K>(
 export function trimLastForwardSlash(url: string) {
   return url.replace(/\/$/, '')
 }
+
+export function isLocalLink(href?: string | null) {
+  return (
+    typeof href === 'string' && !href.startsWith('https://') && !href.startsWith('http://') && !href.startsWith('//')
```
Comment (1emu) [inline src/clients/utils.ts:39]: I think we could write this like:

```typescript
return typeof href === 'string' && !/^((https?):)?\/\//.test(href)
```


image

**Review by 1emu — APPROVED** [review]

**Review by andyesp — APPROVED** [review]

**Pull request merged**